curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-and-share-about-educational-and-professional-background/657b153ac677705c7059530d.md

```diff
@@ -0,0 +1,62 @@
+---
+id: 657b153ac677705c7059530d
+title: Task 20
+challengeType: 19
+dashedName: task-20
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Anna: Hello! Thanks for coming in today. Could you tell me about your previous roles and responsibilities?
+-->
+
+# --description--
+
+The phrase `could you tell me` is a polite way to ask someone for information. It's commonly used in formal or professional situations, like job interviews or meetings.
+In English, some verbs have irregular past forms, meaning they do not follow the regular pattern of adding `-ed` to make the past tense. `Can` is one such verb, and its irregular simple past form is `could`. 
+
+The existence of irregular forms, like `could`, often comes from the way language has evolved over time. These forms do not follow the regular patterns and need to be memorized. 
+
+For example:
+
+- Present: `I can speak English.`
+- Past: `I could speak English when I was six.`
```

Comment:
```suggestion
The existence of irregular forms, like `could`, often comes from the way language has evolved over time. These forms do not follow the regular patterns and need to be memorized. For example:
Present: `I can speak English.` 
Past: `I could speak English when I was six.`
```

Reply:
![image](https://github.com/freeCodeCamp/freeCodeCamp/assets/15801806/116f4cb2-bb7a-45e6-9af2-3305c024af90)

Is this what we want? If so:

```suggestion
The existence of irregular forms, like `could`, often comes from the way language has evolved over time. These forms do not follow the regular patterns and need to be memorized.  For example:

Present: `I can speak English.`

Past: `I could speak English when I was six.`
```

Reply:
It's possible to squish them together:
![image](https://github.com/freeCodeCamp/freeCodeCamp/assets/15801806/bce8de51-c4a4-4a37-afa9-8f5c06a9a1b4)

let me know which you want.